src/ipv6.rs

```diff
@@ -20,29 +23,39 @@ pub struct IPv6Header {
     pub dest_addr: Ipv6Addr,
 }
 
+// To remove ?
```

Comment:
I think we should not make this function public, this is likely to be use only by us, in fact I would prefer inline this unwrap call.